src/stores/GithubStore.js

```diff
@@ -21,22 +73,13 @@ export default class GithubStore {
     }
   }
 
-  async getValue(key) {
-    const pathParts = key.split('.');
-    if (this.config.rootPath) {
-      pathParts.unshift(this.config.rootPath);
-    }
-    let path = pathParts.join('/');
-    if (this.config.fileSuffix) {
-      path += '.md';
-    }
-
+  async getValueFromPath(objectPath, decode = true) {
```

Comment:
Two things here:
1. Should this return a `Stream`? Or does Buffer also avoid forcing the entire file into memory?
2. Maybe this should be named something like `streamBinaryFromPath` to make clear that this is for getting raw binary data from the store.